vector/src/main/java/im/vector/app/features/spaces/invite/SpaceInviteBottomSheetViewModel.kt

```diff
@@ -57,6 +61,37 @@ class SpaceInviteBottomSheetViewModel @AssistedInject constructor(
                         peopleYouKnow = Success(peopleYouKnow)
                 )
             }
+
+            if (roomSummary.membership == Membership.INVITE) {
```

Comment:
Can you remove some nesting because it's hard to read here!

Reply:
Done, extracted a new private function do refresh summary in case of invite